umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -152,15 +152,15 @@
   for SwathWidth."
   [collection]
   (let [swath-width-unit (get-in collection [:SpatialExtent :OrbitParameters :SwathWidthUnit])
-        swath-width (get-in collection [:SpatialExtent :OrbitParameters :SwathWidth]) 
+        swath-width (get-in collection [:SpatialExtent :OrbitParameters :SwathWidth])
         swath-width (if (and swath-width (= "Meter" swath-width-unit))
-                      (/ swath-width 1000)
+                      (double (/ swath-width 1000))
                       swath-width)]
     (if swath-width
       swath-width
       ;; if SwathWidth doesn't exist, Footprints is required.
       (get-largest-footprint-in-kilometer (get-in collection [:SpatialExtent :OrbitParameters :Footprints])))))
```

Comment:
Need to do the same in get-largest-footprint-in-kilometer, when FootprintUnit is Meter.

Reply:
I have made the change to get-largest-footprint-in-kilometer